beast/physicsmodel/tests/test_dust_prior_weights.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+import pytest
+
+#from beast.physicsmodel.prior_weights_dust import set_av_weights
+from beast.physicsmodel.prior_weights_dust import PriorWeightsDust
+
+def test_dust_prior_weights():
+    """
+    Test the dust prior weights
+    """
+
+    av_vals = [0., 2., 4., 6., 8., 10.]
+    rv_vals = [2., 3., 4., 5., 6.]
+    fA_vals = [0.0, 0.2, 0.4, 0.6, 0.8, 1.0]
+    av_model = {"name":"flat"}
+    rv_model = {"name":"flat"}
+    fA_model = {"name":"flat"}
+
+    prior_weights = PriorWeightsDust(av_vals, av_model, rv_vals, rv_model, fA_vals, fA_model)
+
+    # test if priors are set correctly when creating an object of the class PriorWeightsDust
+    np.testing.assert_allclose(prior_weights.av_priors,np.full((len(av_vals)), 1.0))
```

Comment:
Would be great to add an error message (using the err_msg keyword) to make it a little easier to know what happened when the assert fails. 
Same comment for all the assert statements.

Reply:
Done. Are these messages what you had in mind?